numpy/core/src/multiarray/ctors.c

```diff
@@ -1259,6 +1259,52 @@ PyArray_New(PyTypeObject *subtype, int nd, npy_intp *dims, int type_num,
     return new;
 }
 
+int
+_copy_from_pil_style(char* dest, int ndim, char *buf, npy_intp *shape,
+                     npy_intp *strides, npy_intp *suboffsets, npy_intp itemsize)
+{
+    int ax, last_axis = 0;
```

Comment:
It may be good to add a comment like `/* setting last_axis to anything to silence compiler warnings */` so that no one ends up spending long hours trying to figure out why `0` and not the seemingly more reasonable `-1`.